src/test/java/org/broadinstitute/hellbender/utils/nio/XorWrapper.java

```diff
@@ -0,0 +1,80 @@
+package org.broadinstitute.hellbender.utils.nio;
+
+import com.google.common.annotations.VisibleForTesting;
+import java.io.IOException;
+import java.nio.ByteBuffer;
+import java.nio.channels.NonWritableChannelException;
+import java.nio.channels.SeekableByteChannel;
+import java.util.function.Function;
+
+/**
+ * A SeekableByteChannel wrapper for testing.
+ */
+public class XorWrapper implements SeekableByteChannel {
```

Comment:
We generally put test-related utility classes in `src/main/java/org/broadinstitute/hellbender/utils/test/`, rather than under `src/test/java`, which we generally reserve for tests themselves.